vignettes/academictwitteR-batchcompliance.Rmd

```diff
@@ -0,0 +1,64 @@
+---
+title: "Batch Compliance"
+author: Christopher Barrie, Justin Ho, Chung-hong Chan
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Batch Compliance}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+## Introduction
+
+It is crucial for Twitter data holders to ensure the data reflects user intent and the current state of content on Twitter. Specifically, if a tweet is deleted or modified on Twitter, data holders must delete or modify any content stored offline accordingly. The batch compliance endpoints and corresponding functions offer researchers an easy way to help maintain Twitter data in compliance with the [Twitter Developer Agreement and Policy](https://developer.twitter.com/en/developer-terms/policy).
+
+## Upload Dataset
+
+To create a compliance job and upload a dataset, you can use the `create_compliance_job()` function:
+
+```{r, eval = FALSE}
+create_compliance_job(x = "tweet_ids.txt",
+                      type = "tweets")
+```
+
+You can either supply a vector of tweet/user ids or a text file in which each line contains a Tweet ID or user ID. The contents of the text file can look something like this:
```

Comment:
@justinchuntingho It cannot look like something else, or it shouldn't be "can".

This is a factual statement and so make it discrete. I think this should be "should"; or even "The content of the text file looks like this".